src/Form/grid/Element/index.tsx

```diff
@@ -158,7 +158,8 @@ const Element = ({ node: el, form, flags }: any) => {
         if (isFieldActuallyRequired(element, activeStep)) {
           const servar = element.servar;
           let fieldVal: any = fieldValues[servar.key];
-          if (servar.repeated) fieldVal = fieldVal[repeat];
+          if (servar.repeated && repeat !== undefined)
```

Comment:
doesn't servar.repeated = True imply repeated isn't undefined?

Reply:
I think so, but wanted to be explicit on the check.

Reply:
Ok, not generally the way I like to code, but made the change.